packages/styles/src/components/form-check.scss

```
@@ -9,14 +9,29 @@
 @use '../mixins/color' as color-mx;
 @use '../mixins/icons' as icons-mx;
 @use '../mixins/utilities' as utility-mx;
+@use '../tokens/components' as components;
+@use '../functions/tokens' as tokens;
```

This is not necessary:
```suggestion
@use '../tokens/components';
@use '../functions/tokens';
```